### PR TITLE
hhristov/recycle native view android

### PR DESCRIPTION
**Pull request by hshristov**
Rename:
_createNativeView to createNativeView;
_initNativeView to initNativeView
_disposeNativeView to disposeNativeView
_resetNativeView to resetNativeView

### CODE REVIEW

**Comment by hshristov** [comment]
run ci

**Comment by hshristov** [comment]
uitests

File: tns-core-modules/ui/time-picker/time-picker.android.ts [hunk 1]
```diff
@@ -22,80 +24,73 @@ function initializeTimeChangedListener(): void {
...
         onTimeChanged(picker: android.widget.TimePicker, hour: number, minute: number): void {
             const timePicker = this.owner;
-            let validTime = getValidTime(timePicker, hour, minute);
-            timePicker._setNativeValueSilently(validTime.hour, validTime.minute);
+            if (timePicker.updatingNativeValue) {
+                return;
+            }
+
+            const validTime = getValidTime(timePicker, hour, minute);
+            hourProperty.nativeValueChange(timePicker, validTime.hour);
```
Comment (vakrilov) [inline tns-core-modules/ui/time-picker/time-picker.android.ts]: delete those - not needed

**Comment by vchimev** [comment]
widgets

**Comment by vchimev** [comment]
ci/jenkins/widgets — PASSed.

**Comment by vchimev** [comment]
widgets-uitests-android

**Comment by vchimev** [comment]
widgets-uitests-ios

**Comment by vchimev** [comment]
widgets

**Comment by vchimev** [comment]
widgets-uitests-android

**Comment by vchimev** [comment]
widgets-uitests-ios

**Comment by vchimev** [comment]
widgets

**Comment by vchimev** [comment]
widgets

**Comment by vchimev** [comment]
@hshristov,

there are some failures for iOS that require you to look into them.

**Pull request merged**